packages/amplify-cli-core/API.md

```diff
@@ -232,6 +232,10 @@ export class AngularConfigNotFoundError extends Error {
 
 // @public (undocumented)
 export class ApiCategoryFacade {
+    // (undocumented)
+    static addGraphQLAuthorizationMode(context: $TSContext, authType: string): Promise<void>;
+    // (undocumented)
+    static generateContainersArtifacts(context: $TSContext, resource: $TSAny): Promise<$TSObject>;
     // (undocumented)
```

Comment:
does this and below depend in any way on `@aws-amplify/graphql-transformer-core` or `graphql-transformer-core` ?

Reply:
No new dependencies, there's an implicit dep here on the api category, but nothing where we're adding the transformer-core deps directly.

Reply:
I think that in long term we should find better place to host plugin Facades. Perhaps separate package. Core should be leaf dependency and just bring utilities.